include/mrdox/Metadata/Javadoc.hpp

```diff
@@ -65,6 +65,18 @@ struct MRDOX_VISIBLE
         warning
     };
 
+    /** Parameter pass direction.
+    */
+    enum class ParamDirection : int
+    {
+        // KRYSTIAN NOTE: no idea why this is needed,
+        // does not seem to break when changed to 0
+        none = 1, // needed by bitstream
```

Comment:
does that mean unspecified?

Reply:
I think it does and I think we can use 0 here